data/brands/tourism/hotel.json

```diff
@@ -1734,6 +1734,16 @@
         "tourism": "hotel"
       }
     },
+    {
+      "displayName": "Paradores",
+      "locationSet": {"include": ["es", "pt"]},
```

Comment:
[This Wikidata item](https://www.wikidata.org/wiki/Q1476845) only covers the Spanish chain. Should Portugal should get a separate entry linked to [Pousadas de Portugal](https://www.wikidata.org/wiki/Q2186274)?

Reply:
No. That's a different brand. I've included portugal since [their website](https://www.parador.es/) include a hotel in Portugal. Perhaps, it doesn't worth it :thinking:, what do you think?

Reply:
OK, if they’re in Portugal too then this is correct.

Reply:
In fact, let me add the Pousadas item as well, killing two birds with one stone :v:

Reply:
Done